main.js

```diff
@@ -7,39 +7,42 @@ var exerciseButton = document.querySelector('.exercise');
 var accomplishInput = document.querySelector('.accomplish-input');
 var minutesInput = document.querySelector('.minutes-input');
 var secondsInput = document.querySelector('.seconds-input');
-
+var buttonText = document.querySelectorAll('.cat-titles')
+var newActivitySection = document.querySelector('.new');
+var currentActivitySection = document.querySelector('.current');
+var startButton = document.querySelector('.start-button');
 var startActivityBtn = document.querySelector('.start-activity-button');
 
 studyButton.addEventListener('click', changeStudyColor);
 meditateButton.addEventListener('click', changeMeditateColor);
 exerciseButton.addEventListener('click', changeExerciseColor);
 minutesInput.addEventListener('keyup', limitMin);
 secondsInput.addEventListener('keyup', limitSec);
-
 startActivityBtn.addEventListener('click', createCurrentActivity);
 
-function changeColor(button, category) {
-    button.classList.add(category);
-}
-
-function returnDefaultColor(button1, category1, button2, category2) {
-    button1.classList.remove(category1);
-    button2.classList.remove(category2);
+function changeColor(button1, category1, button2, category2, button3, category3) {
```

Comment:
Great job getting this to work, I see we needed an option for if only one button is defined